affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleDetectorPlugin.kt

```diff
@@ -166,18 +163,22 @@ class AffectedModuleDetectorPlugin : Plugin<Project> {
         testType: AffectedModuleTaskType,
         project: Project
     ) {
+        val affectedPath = getAffectedPath(testType, project)
+        val configuration = AffectedModuleDetector.getRootConfiguration(project)
+        if (affectedPath == null || AffectedModuleDetector.isModuleExcluded(configuration, project)) {
+            return
```

Comment:
The main line of this fix. We can skip calling `.dependsOn` and `.afterEvaluate` for excluded modules at all. I hope it will fix the problem of our colleague from Peru. 

Because I guess, if the module has not a Detekt but one was called with .dependsOn, Detekt might try calling any deps, try to configure smth, and create a crash per issue 159.